libcxx/test/std/containers/sequences/array/array.cons/initialization.pass.cpp

```diff
@@ -18,26 +18,33 @@ struct NoDefault {
     TEST_CONSTEXPR NoDefault(int) { }
 };
 
-// Test default initialization
-// This one isn't constexpr because omitting to initialize fundamental types
-// isn't valid in a constexpr context.
-struct test_default_initialization {
+struct test_initialization {
```

Comment:
Please make these separate tests rather than trying to reuse the instantiation. It's just clearer for readers.

Reply:
The only reason this was multiple structs previously is basically that I couldn't run the default initialization test in constexpr, so I needed another named struct for it. The way this is written right now follows our usual pattern of having something like:

```
constexpr bool test() {
  // test case 1
  {
    ...
  }
  // test case 2
  {
    ...
  }
  // test case 3
  {
    ...
  }

  return true;
}

int main() {
  test();
  static_assert(test());
}
```